build.gradle

```diff
@@ -458,6 +457,7 @@ test {
 
 //add gatk launcher script to the jar as a resource
 processResources {
+    dependsOn(downloadGsaLibFile)
     from("gatk")
```

Comment:
Can you explain why adding the launcher script to the jar depends on downloading GSA lib, which is an R library?

Reply:
That's a reasonable question.  I would say it's because that comment doesn't apply to the following line anymore and I should have moved it...

Reply:
Fixed.  ProcessResources depends on the R library download because the R library is one of the resources we want in the jar.  It is not however the launcher script.